CHANGELOG

```diff
@@ -60,6 +60,9 @@ Linear and nonlinear solvers
   sides, block low-rank compression, builds using 64-bit integers, and other
   improvements.
 
+- Added an interface to the MKL Pardiso solver, a shared-memory multiprocessing
+  parallel direct sparse solver developed by Intel.
```

Comment:
Thanks for adding this.

The comment for `MKL CPardiso` from mfem-4.2 was:

> Added an interface to the MKL CPardiso solver, an MPI-parallel sparse direct
> solver developed by Intel. See Example 11p for an illustration of its usage.

The above sounds very similar to it, can we clarify what the difference between the two solvers is in the above comment?

cc: @jandrej, @psocratis

Reply:
How about

> Added an interface to the MKL Pardiso solver, a shared-memory multiprocessing
> parallel direct sparse solver developed by Intel. This interface provides a serial version of Pardiso for use with `SparseMatrix`.

Reply:
How does `multiprocessing parallel` connect  with `SparseMatrix`, which is serial?

Reply:
```suggestion
- Added an interface to the MKL Pardiso sparse direct solver developed by Intel.
  This interface provides a serial (OpenMP shared memory) version of Pardiso for
  use with SparseMatrix. This complements the existing parallel (MPI distributed
  memory) version already available through the CPardiso MFEM integration.
```

@adam-sim-dev and @jandrej -- are you OK with this rewording?

Reply:
Looks good to me.

Reply:
👍